templates/terraform-deploy-usage-template.yaml

```diff
@@ -0,0 +1,16 @@
+jobs:
```

Comment:
This is not a complete template from start to finish (doesn't include e.g. a trigger), rather it is an example of usage. We can move it (alongside the rest of workflows), but it will still not comply with specification from that link.

Reply:
In that case, can we move it to the `examples` instead of templates? WDYT? Or at least mention that it's more an example than template.

Reply:
Done